File: module-common/src/main/java/com/geekhome/common/configurable/StringField.java
package com.geekhome.common.configurable;

import com.geekhome.common.localization.Resource;

public class StringField extends FieldDefinition<String> {

    public StringField(String name, Resource hint, int maxSize, Validator<String>... validators) {
        super(name, hint, maxSize, String.class, new StringFieldBuilder(), validators);
    }
}

File: module-domain/src/main/java/eu/geekhome/domain/configurable/StringField.kt
package eu.geekhome.domain.configurable;

import eu.geekhome.data.localization.Resource;

public class StringField extends FieldDefinition<String> {

    public StringField(String name, Resource hint, int maxSize, Validator<String>... validators) {
        super(name, hint, maxSize, String.class, new StringFieldBuilder(), validators);
    }
}
